src/components/footer/ChatContacts.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+import { RouteComponentProps } from "@reach/router";
+import Phone from "../icons/Phone";
+import Email from "../icons/Email";
+
+const InlineDiv = styled.div`
+  display: inline-block;
+`;
+
+const ContactsContainer = styled.div`
```

Comment:
you don't need to write "Contacts" It's ChatContact component and the component is used only here so container for naming is good